timetracker/entries/migrations/0003_load_client_data.py~

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""this file was created using the following command: 
+    python manage.py makemigrations --empty historical_data"""
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+"""This method insert_data is used to add data to Client model 
+using the information from model Project"""
+def insert_data(apps, schema_editor):
+    Project = apps.get_model('entries', 'Project')
+    Client = apps.get_model('entries', 'Client')
+
+    # create query_set object to filter entries without client
+    #query_set = Project.objects.filter(client)
+
+    for project in Project.objects.all():
+        # Get the client name from project model and update
+        # the name attribute of client
+        client = Client(name=project.client)
```

Comment:
This looks good.  The only issue here is when you have multiple Projects with the same Client value, you'll wind up with multiple Clients with the same name.

To avoid this we would use [get_or_create](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#get-or-create)